### PR TITLE
Rename peripheral model classes

### PR DESCRIPTION
**Pull request by j9liu**
This PR deletes unneeded classes related to old `Model`, and renames the remaining `ModelExperimental` classes. Additionally, the `ModelExperimental` folders have been renamed to `Model`.

@ptrgags when you get the chance, can you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @j9liu!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Source/Scene/Model/CustomShaderGuide/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## :warning: Custom Shaders documentation has moved! :warning
+# :warning: Custom Shaders documentation has moved! :warning
```
Comment (ptrgags) [inline Source/Scene/Model/CustomShaderGuide/README.md:1]: I think we can remove this file now

**Review by ptrgags — COMMENTED** [review]
@j9liu the renaming looks good! I swept through the old `Model.js`'s imports to make sure all the helper functions have been accounted for.

I just had one small comment before I merge this

**Comment by ptrgags** [comment]
Thanks @j9liu!

**Pull request merged**